deploy/terraform-datacommons-website/examples/website_v1/main.tf

```diff
@@ -66,6 +66,7 @@ module "apikeys" {
 module "esp" {
   source                   =  "../../modules/esp"
   project_id               = var.project_id
+  mixer_githash            = var.mixer_githash
```

Comment:
is website_gitbash needed?

Reply:
nope, only mixer_githash is needed to fetch the versioned mixer grpc pb.